src/index.js

```diff
@@ -348,28 +355,43 @@ class JanusAdapter {
     return Date.now() + this.avgTimeOffset;
   }
 
-  getMediaStream(clientId) {
+  getMediaStream(clientId, type = 'audio') {
```

Comment:
Technically this should default to "both" instead of just "audio", but that would require a change to `networked-aframe` as well.